src/shared/components/BlockList/BlockList.tsx

```diff
@@ -79,6 +79,7 @@ const BlockList: FC<BlockListProps> = ({ blocks, config }) => {
 						pastDeadline={config?.ZOEK?.pastDeadline || false}
 						onSearchButtonClicked={config?.ZOEK?.onSearchButtonClicked || noop}
 						onCollectionButtonClicked={config?.ZOEK?.onCollectionButtonClicked || noop}
+						educationLevel={config?.ZOEK?.educationLevel || undefined}
```

Comment:
This will automatically be undefined if it is undefined. Or can this be an empty string of 0?
```suggestion
						educationLevel={config?.ZOEK?.educationLevel}
```